front/src/__tests__/util_tests/getColorForShowIds.test.ts

```diff
@@ -0,0 +1,22 @@
+import { getUniqueColorsForShowIds } from '../../utils/getColorForShowId';
+
+describe('getUnqueColorsForShowIds', () => {
+
+  it('returns 2 unique colors', () => {
+    const colors = getUniqueColorsForShowIds([1, 2]);
+    expect(colors[0]).not.toEqual(colors[1]);
+  });
+
+  it('returns 4 unique colors', () => {
+    const colors = getUniqueColorsForShowIds([1, 2, 3, 4]);
+    expect(colors[0]).not.toEqual(colors[1]);
+    expect(colors[2]).not.toEqual(colors[3]);
```

Comment:
- Could we use IDs from 1,000-100,000? Something like this `[23455, 33455, 33422, 1432, 5699]`  
- Instead of checking against one item, could something like this work to test that each item is unique?

https://stackoverflow.com/a/57002833/9830506

Reply:
Yep